kytos/core/controller.py

```diff
@@ -656,7 +656,7 @@ async def msg_out_event_handler(self):
                                    message.header.message_type,
                                    message.header.xid,
                                    packet.hex())
-                    self.notify_listeners(triggered_event)
+                    await self.notify_listeners(triggered_event)
```

Comment:
This should also be inside of  `self.loop.create_task()`. I am going to fix this when fixing the unit tests

Reply:
No longer using async.